src/GraphQL/loaders/classSchemaQueries.js

```diff
@@ -0,0 +1,86 @@
+import Parse from 'parse/node';
+import { GraphQLNonNull, GraphQLList } from 'graphql';
+import { transformToGraphQL } from '../transformers/schemaFields';
+import * as classSchemaTypes from './classSchemaTypes';
+import { enforceMasterKeyAccess } from '../parseGraphQLUtils';
+
+const getClass = async (name, schema) => {
+  try {
+    return await schema.getOneSchema(name, true);
+  } catch (e) {
+    if (e === undefined) {
```

Comment:
I think we must address this undefined error in a second moment, sounds really weird.

Reply:
Yes.. but it came from the legacy code and it is actually needed to make it work with the current adapter implementation.